### PR TITLE
completed ticket 8 -> added a patch request to update the number of v…

### PR DESCRIPTION
**Pull request by zahraakarim**
…otes of an article when given an article id

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -273,3 +273,73 @@ describe("POST /api/articles/:article_id/comments", () => {
...
+  });
+  test("404: ERROR: responds with an error when article_id type is valid but doesn't exist", () => {
+    return request(app)
+      .patch("/api/articles/9999")
+      .send({ inc_votes: 27 })
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Path not found");
+      });
+  });
```
Comment (johnhutcheon) [inline __tests__/app.test.js:353]: Really excellent testing so far Zahraa! Can I just request one more though please? Could you please add a test for when inc_votes is not a number? I.e. "inc_votes: nonsense"

**Review by johnhutcheon — CHANGES_REQUESTED** [review]
Great work Zahraa, your testing is excellent and you're considering a lot of outcomes which is great to see! 🙌 
Could you just add the further test I've mentioned please? Thanks 🙂

File: __tests__/app.test.js (hunk 1 above)
Comment (zahraakarim) [reply __tests__/app.test.js:353]: Of course, will add in now, thank you!

**Review by pippaamy — APPROVED** [review]
Well done, great changes! go ahead and merge :)

**Pull request merged**